Mod2/Project/index.md

```diff
@@ -0,0 +1,76 @@
+# Message Logger
+
+## Learning Goals
+- Demonstrate:
+    * Version Control, including branches and pull requests.
+    * Understanding of database table relationships.
+    * Database CRUD with an ORM.
+
+- Explore:
+    * Dive into a new (not written by you) code base!
+    * Using TDD to add additional features!
+
+
+## Project Description
+the aim of this project is to build on the Message Logger we created in Mod1.  Now, a user should be able exit the application, and return later to review past mesages.  This will require us to add a database to our Message Logger projects.
+
+This project will give you the opportunity to showcase all of the concepts we have learned this Mod.  There might be skills that we covered all the way back in week 1, so don't forget to go back and review those earlier lessons and labs!
+
+**Project Setup**
+To get started on this project:
+
+1. **fork and clone** this [starter repo](https://github.com/turingschool-examples/LaunchMod2Project)
+    * The starter repo has code that will be similar to your Mod1 Project.
+2. Run all tests to make sure the project is working (all tests should be passing).
+
+When you have completed the Project Setup, send a link to your cloned repository to all your instructors.
+
+
+### Iteration 1 - Creating and Conecting a Database
+In order to save messages and users so that they persist when a user exits and restarts the program, we will need to create a database for our Message Logger application.
+
+You should create a branch for your work on this iteration.
+
+* Review the classes that exist in the starter program - determine how these classes will be reflected in the tables of a database.  Be sure to think about what relationships may exist.
+* Use the Npgsql.EntityFrameworkCore.PostgreSQL Nuget package to create a database context in your application.
+* Update your Message and User classes to be used as a basis for your database tables.
+
+When you are finished with iteration 1, create a pull request and schedule time with a cohort-mate for some code review.
```

Comment:
I like this code review as part of the project! I wonder if there is some sort of deliverable tied to this, maybe in the form where they turn in the project they need to also turn in the three most useful pieces of feedback they incorporated?

Reply:
Yes, love the idea of a deliverable... maybe some comments on the PR to mimic developer communication?

Reply:
Love that idea!